tc/core/tc_executor_new_api-inl.h

```diff
@@ -0,0 +1,148 @@
+/**
```

Comment:
This file doesn't appear to be included from anywhere.

Reply:
Files are living side-by-side and I will run git mv -f as part of the last commit.